src/main/java/calculator/view/InputView.java

```diff
@@ -0,0 +1,13 @@
+package calculator.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static Scanner scan = new Scanner(System.in);
```

Comment:
정적 상수로 선언되도 좋을 것 같습니다. 객체 자체는 재사용 해도 괜찮을 것 같아요 🙇 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```